src/simple_tracker_launch/launch/simple_tracker_launch.py

```diff
@@ -17,14 +17,13 @@
 from ament_index_python.packages import get_package_share_directory
 
 def generate_launch_description():
-
-    config = os.path.join(
-        get_package_share_directory('simple_tracker_launch'),
-        'config',
-        'params.yaml'
-        #'params-cuda.yaml'
-        #'params-mike.yaml'
-    )
+    
+    #video_file = os.path.join(get_package_share_directory('simple_tracker_launch'), 'videos', 'plane_flying_past2.mkv')
```

Comment:
I worry about checked in data as the path forward.

A problem for another day perhaps, but worth noting.
Eventually it doesn't scale to keep data in git.

Reply:
That is a very fair point, in the old tracker we also could specify stuff using environment variables as well as secrets files. ROS uses a different configuration system, but yes, ideally it would be better driven in a different way versus this sort of specific config file approach.